optimum/habana/accelerate/accelerator.py

```diff
@@ -220,6 +221,11 @@ def __init__(
             **kwargs,
         )
 
+        if self.state.is_fp8_enabled:
+            if self.fp8_recipe_handler is None:
+                self.fp8_recipe_handler = GaudiFP8RecipeKwargs()
```

Comment:
Why do we need this init? Shouldn't this be already initialized in  L205 above.

Reply:
Setting with default values if user doesn't specify anything.